recipes/shenfun/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set build = 1 %}
+{% set name = "shenfun" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/spectralDNS/shenfun/archive/{{ version }}.tar.gz
+  sha256: 48e51882b2d45359997aee4de3d7dc46c81db2ee46b5ddfb330ebdb81ba312ad
+
+build:
+  number: {{ build }}
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - numpy
+    - setuptools
+    - python
+    - cython
+    - fftw
+    - pip
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - cython
+    - scipy
+    - fftw
+    - mpi4py
+    - mpi4py-fft
+    - {{ mpi }}
```

Comment:
If the build only uses `mpi4py` at runtime, you don't need this

Reply:
But I use mpi for hdf5 and h5py, I need the mpi variant there, don't I?

Reply:
You can just have `- h5py`

Reply:
Would it pick up the mpi version without using mpi_{{ mpi }}_*.